packages/control-property-editor/src/panels/changes/ConfigGroup.tsx

```diff
@@ -0,0 +1,70 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import { Stack, Text } from '@fluentui/react';
+
+import type {
+    PendingConfigurationChange,
+    SavedConfigurationChange
+} from '@sap-ux-private/control-property-editor-common';
+import { SAVED_CHANGE_TYPE } from '@sap-ux-private/control-property-editor-common';
+
+import styles from './ControlGroup.module.scss';
+import { ConfigChange as ConfigurationChange } from './ConfigChange';
+
+export interface ConfigGroupProps {
+    text: string;
+    configPath: string;
+    index: number;
+    changes: (PendingConfigurationChange | SavedConfigurationChange)[];
```

Comment:
We can't have mixed changes in the array. It is either `PendingConfigurationChanges[]` or `SavedConfigurationChange[]`.

Reply:
adapted

Reply:
This for the change stack in the UI while converting changes https://github.com/SAP/open-ux-tools/blob/a84038875f8f4dffeb60fbe0e309300a3d3f96e8/packages/control-property-editor/src/panels/changes/ChangeStack.tsx#L154